src/CourseDashboard.hpp

```diff
@@ -12,6 +12,8 @@ class CourseDashboard
     void deleteUserByNick(std::string nick);
     void retriveUserByNick(std::string nick);
     void updateUser(User & user);
+    void saveToFile(User & user);
+    void loadFromFile();
```

Comment:
loadFromFile loads all users, but saveToFile saves only one. How to save all users?

Reply:
Currently all users are provided into file.